src/Features/Core/Portable/EditAndContinue/EditSession.cs

```diff
@@ -850,6 +860,13 @@ public async ValueTask<SolutionUpdate> EmitSolutionUpdateAsync(Solution solution
                     continue;
                 }
 
+                // We don't track document changes in stale projects until they are rebuilt (removed from stale set).
+                if (oldSolution.IsStaleProject(newProject.Id))
+                {
+                    Log.Write($"EnC state of {newProject.Name} '{newProject.FilePath}' queried: project is stale");
```

Comment:
i assume these logs are for us.  so this seems fine.  it feels like maybe we want to say more than "stale".  perhaps "contains a document that was changed but was not rebuilt due to SingleTargetBuildForStartupProjects "

Reply:
These are logs for diagnostic purposes only. One needs to be familiar with the implementation to reason about most of the detailed log messages.

Reply:
wfm!